app/src/main/java/de/westnordost/streetcomplete/quests/wheelchair_access/AddWheelchairAccessToilets.kt

```diff
@@ -28,7 +28,10 @@ class AddWheelchairAccessToilets : OsmFilterQuestType<WheelchairAccess>() {
 
     override fun createForm() = AddWheelchairAccessToiletsForm()
 
-    override fun applyAnswerTo(answer: WheelchairAccess, tags: Tags, geometry: ElementGeometry, timestampEdited: Long) {
-        tags.updateWithCheckDate("wheelchair", answer.osmValue)
+    override fun applyAnswerTo(answer: WheelchairAccessToiletsAnswer, tags: Tags, geometry: ElementGeometry, timestampEdited: Long) {
+        when (answer) {
+            is WheelchairAccessToilets -> answer.applyTo(tags)
+            is NoToilet -> tags.updateWithCheckDate("toilets", "no")
+        }
```

Comment:
Is it wanted that this now uses `toilets:wheelchair` instead of `wheelchair` for `amenity=toilets`?
Also adding `toilets=no` here doesn't really make sense.

Reply:
> Edit: toilets:wheelchair [should not be used here](https://wiki.openstreetmap.org/wiki/Tag:amenity=toilets#Toilets_within_places)

Are you sure? 

> Facilities such as shops and pubs often have toilets inside them for customers, and sometimes free for the public

> In general please do not map toilets that are inaccessible to the public. Many buildings have toilets inside for workers or owners

toilets:wheelchair seems applicable to customer-only toilets

Reply:
Still it might be a good idea to not go against the vast majority of current tagging:
`amenity=toilets` with `wheelchair`: 74 887
`amenity=toilets` with `toilets:wheelchair`: 7 124

Reply:
No. That was introduced here when applyTo was introduced. I'll revert back to before that this evening